### PR TITLE
Sutton Armour - Offline version so my exporter works

### PR DESCRIPTION
**Pull request by Erdertus**
Add Sutton Armor sponsored by Argoth, copied locally to own branch as exporter wasn't working on Meows PR.

### CODE REVIEW

**Review by Meowkov — APPROVED** [review]
Looking good

File: weapon_descriptions.xml [hunk 1]
```diff
@@ -11347,7 +11347,7 @@
       <WeaponFlag value="TwoHandIdleOnMount" />
     </WeaponFlags>
     <AvailablePieces>
-    <AvailablePiece id="crpg_polesword_handle_h0" />
+      <AvailablePiece id="crpg_polesword_handle_h0" />
```
Comment (Yeldur) [inline weapon_descriptions.xml:11350]: Intended?

Comment (Yeldur) [reply weapon_descriptions.xml:11350]: spoke with meow who confirmed non-issue, appears to be an auto-fixed whitespace

**Review by Yeldur — APPROVED** [review]
Approved

**Review by Muparadzi — APPROVED** [review]

**Pull request merged**